pkg/network/protocols/http/monitor.go

```diff
@@ -101,6 +105,17 @@ func (m *Monitor) Start() error {
 	return m.processMonitor.Initialize()
 }
 
+func (m *Monitor) GetUSMStats() map[string]interface{} {
+	if m == nil {
+		return map[string]interface{}{
+			"Error": USMStartupError.Error(),
```

Comment:
Consider changing it to `HTTPError` or something similar, so we could easily add other protocols to the status in the near future

Reply:
the monitor should be renamed and instead of http monitor it should be USMMonitor or something similar
@p-lambert asked to wait until CO-RE support is merged before making the rename